pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -77,6 +77,13 @@ class PostProcessor(Post):
     app :
         Inherited parent object.
 
+    Examples
+    --------
+    Basic usage demonstrated with an HFSS, Maxwell or any other design:
```

Comment:
```suggestion
    Basic usage demonstrated with an HFSS, Maxwell, or any other design:
```

Reply:
Ansys doc style is to use the Oxford comma, which is a comma before the "and" or the "or" in a series of items.